src/shared_modules/utils/stringHelper.h

```diff
@@ -25,6 +25,42 @@
 
 namespace Utils
 {
+    static void ISO8859ToUTF8(std::string& data)
+    {
+        // Convert from ISO-8859-1 to UTF-8
+        std::string strOut;
+        // 0xc0 is 11000000 in binary, used to check if the character is a 2-byte sequence
+        constexpr auto UTF8_2BYTE_SEQ{ 0xc0 };
+        // 6 is the number of bits to shift the character to the right
+        constexpr auto UTF8_2BYTE_SEQ_VALUE_LEN{ 6 };
+        // 0x80 is 10000000 in binary, used to check if the character is a 2-byte sequence
+        constexpr auto UTF8_2BYTE_FIRST_CODE_VALUE{ 0x80 };
+        // 0x3f is 00111111 in binary, used to mask the last 6 bits of the character
+        constexpr auto UTF8_2BYTE_MASK{ 0x3f };
+
+        for (auto it = data.begin(); it != data.end(); ++it)
+        {
+            const uint8_t ch = *it;
+
+            // ASCII character
+            if (ch < UTF8_2BYTE_FIRST_CODE_VALUE)
+            {
+                strOut.push_back(ch);
+            }
+            // Extended ASCII
+            else
+            {
+                // 2-byte sequence
+                // 110xxxxx 10xxxxxx
```

Comment:
In this line, I'd remove the `10xxxxxx` just to be clear that the following sentence will only create the first byte `110xxxxx`